Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Identifier
+    static let identifier = Identifiers.gymWeekCell
```

Comment:
why do we need the identifier here? are we using this cell across multiple view controllers?